src/Components/Footer/Footer.js

```diff
@@ -7,14 +7,22 @@ import { FaServer, FaGithub} from "react-icons/fa";
 export const Footer = () => {
 
     return (
-        <>
+        
           <footer className="social-section">
             <ul className="footer-links">
-              <li className="foot-link"><a target="_blank" href="https://github.com/Atos20/indecision-maker-server"><FaServer className="repo-link"/></a></li>
-              <li className="foot-link"><a target="_blank" href="https://github.com/nicolegooden/indecision-maker"><FaGithub className="repo-link"/></a></li>
+              <li className="foot-link">
+                <a target="_blank" href="https://github.com/Atos20/indecision-maker-server">
```

Comment:
I've seen errors in the console for using `target="_blank"` , wondering if there is a different attribute or value for this attribute that we can utilize.

Reply:
You might be right, I will double-check that.